content/Day-1/_index.md

```diff
@@ -1,43 +1,38 @@
 ---
-title: Effective Communication (Day 1)
+title: 1. Active Listening
```

Comment:
This will be addressed on 9/29 and an update will be provided moving forward. The size of the level 2 headers is not ideal for the layout of the text I would like to modify this to fit the page better